esp/src/index.html

```diff
@@ -67,7 +67,7 @@
                 xhttp.send();
             }
         }
-    </script>
+    </script> -->
```

Comment:
Is this related to ensuring "Open in new tab" still works (IOW the new tab has doesn't need new credentials)?